modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/utils/ImplicitObjectUtils.java

```diff
@@ -81,7 +81,14 @@ public static ResolverSet buildImplicitResolverSet(ParameterizedModel parameteri
         .orElse(null);
   }
 
-  private static boolean canBeUsedImplicitly(ParameterizedModel parameterizedModel) {
+  /**
+   * Returns whether the given model can be used implicitly. That means that all of its parameters which are non optional have
+   * associated a default value.
```

Comment:
`That means that all of its parameters which are non optional have associated a default value.`
makes no sense, only optional parameters can have a default value.